contracts/modules/licensing/LicensingModule.sol

```diff
@@ -95,6 +116,12 @@ contract LicensingModule is AccessControlled, ILicensingModule, BaseModule, Reen
         DISPUTE_MODULE = IDisputeModule(disputeModule);
     }
```

Comment:
Same as above.  need `_disableInitializers()`

Reply:
This one yes, good catch! 236ca90